Tests/MapboxNavigationTests/RouteDurationTests.swift

```diff
@@ -0,0 +1,91 @@
+import XCTest
+import TestHelper
+import Turf
+import MapboxMaps
+@testable import MapboxNavigation
+
+class RouteDurationTests: TestCase {
+    var navigationMapView:NavigationMapView = NavigationMapView(frame: CGRect(origin: .zero, size: .iPhone6Plus))
+    enum RouteDurationAnnotationTailPosition: Int {
+        case left
+        case right
+    }
+    let routeDurationAnnotationsSource = NavigationMapView.SourceIdentifier.routeDurationAnnotationsSource
+    let routeDurationAnnotationsLayer = NavigationMapView.LayerIdentifier.routeDurationAnnotationsLayer
+
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        navigationMapView.removeRouteDurations()
+        super.tearDown()
+    }
+    
+    func setupRouteDurationsLayer(tailPosition: RouteDurationAnnotationTailPosition) throws {
```

Comment:
You should not be using own implementation of `RouteDurationAnnotationTailPosition` and `setupRouteDurationsLayer(tailPosition:)`. Test should be using similar implementations of `NavigationMapView`. If they're private it's fine to make them internal (default access control).